ansible/roles/docker/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
 - apt: name=apparmor
   tags: 
     - build_ami
-- apt: name=docker-engine=1.7.0-0~trusty
+- apt: name=docker-engine=1.9.1-0~trusty
```

Comment:
Yep - just got released - we're hoping to get a chance to look at it soon.  Eric did a bunch of work testing out 1.10 with the new logging drivers, but the performance was poor (and the API logging performance fell apart as well - https://github.com/docker/docker/issues/20586).  I can't actually remember what issues we ran into with 1.9.